pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -2788,7 +2788,7 @@ void FullscreenUI::DrawInterfaceSettingsPage()
 
 	MenuHeading(FSUI_CSTR("On-Screen Display"));
 	DrawIntSpinBoxSetting(bsi, FSUI_ICONSTR(ICON_FA_SEARCH, "OSD Scale"),
-		FSUI_CSTR("Determines how large the on-screen messages and monitor are."), "EmuCore/GS", "OsdScale", 100, 25, 500, 1, "%d%%");
+		FSUI_CSTR("Determines how large the on-screen messages and monitor are."), "EmuCore/GS", "OsdScale", 100, 25, 500, 1, FSUI_CSTR("%d%%"));
```

Comment:
I'm not sure these should be translated. It's literally just a percentage display.

Reply:
Some languages require a space to be added between the value and the percentage sign. That's why percentage values they should be translatable.

Reply:
Okay, sure.